### DISCUSSION TITLE
Why not adding eslint as a devDependency

### DISCUSSION THREAD

**Discussion by huangzl97** [web-infra-dev/modern.js#5040] [Q&A]
I notice that lint script generated from the [template](https://github.com/web-infra-dev/modern-js-examples/blob/3b20e33cd48df885d8d103f553f6c638c5bf5bf5/examples/basic-app/package.json#L19) is:
```     
 "node --max_old_space_size=8192 ./node_modules/eslint/bin/eslint.js --fix --color --cache --quiet"
```
but the `eslint` was not a devDependencies, is there a particular reason why?

**Comment by chenjiahan** [comment]
@KyrieLii can you help to check this
